UT-Alignment-12d-2 (#16)
bSI-InfraRoom/IFC-infra-unit-test: UT-Alignment-12d-2/readme.md

### Intent

IFC 4x3 RC2
This scenario has a simple alignment with left arc in horizontal geometry and a crest arc for vertical geometry
That is, it has 
Civil Horizontal Geometry (HG):

- straight
- an arc with ifc positive radius of curvature
- a straight

Civil Vertical Geometry (VG):

- straight
- a crest arc
- a straight

In the ifc file there is only the ifc semantic definition of the alignment and no ifc geometry.
The Ifc file was generated by 12d Model. 

### Prerequisites

- UT-Alignment-12d-1

### Content

This scenario covers the additional concepts and/or IFC entities:

> - 'IfcAlignmentVertical'
> - 'IfcAlignmentVerticalSegment' with 'PredefinedType=CONSTANTGRADIENT'
> - 'IfcAlignmentVerticalSegment' with 'PredefinedType=CIRCULARARC' for a crest

Reviewer: ```suggestion
- `IfcAlignmentVertical`
- `IfcAlignmentVerticalSegment` with `PredefinedType=CONSTANTGRADIENT`
- `IfcAlignmentVerticalSegment` with `PredefinedType=CIRCULARARC` for a crest
```

> - `IfcAlignmentVerticalSegment` with `PredefinedType=CIRCULARARC' for a crest

Reviewer: ```suggestion
- `IfcAlignmentVerticalSegment` with `PredefinedType=CIRCULARARC` for a crest
```

Forgot one :)

### Supporting files

Following files correspond to this scenario:

| Filename                           | Description                                                                                            |
|------------------------------------|--------------------------------------------------------------------------------------------------------|
| `UT-Alignment-12d-2.ifc`           | the exported content as an IFC file                                                                    |

> | `UT-Alignment-12d-2.png`           | plan and long section view of the alignment, and the segment parameters (with Civil radius and %Grade) |

Reviewer: ```suggestion
| `UT-Alignment-12d-2.png`           | plan and long section view of the alignment, and the segment parameters (with Civil radius and percent Grade) |
```

Sadly, `%` is also a reserved character in LaTeX ...
